### ISSUE TITLE
Roman Numeral Converter: Tests results incorrect

### ISSUE THREAD

**Issue by savourylie** [FreeCodeCamp/FreeCodeCamp#9090]
#### Challenge Name

Roman Numeral Converter

#### Issue Description

The FCC IDE showed that my code didn't pass some of the tests while the tests are passed no problem on my Chrome 53.0.2762.0 canary (64-bit), and on NodeJS (v4.2.3). What's more interesting is that when I used the same code on the Chinese branch of FCC (freecodecamp.cn), it worked perfectly.

#### Browser Information

* Browser Name, Version: Chrome 53.0.2762.0 canary (64-bit)
* Operating System: OS X 10.11.4
* Mobile, Desktop, or Tablet: Desktop

#### Your Code
```js
// If relevant, paste all of your challenge code in here
"use strict";
"use esversion: 6";

class RomanDict {
	constructor() {
		this.M = 1000;
		this.D = 500;
		this.C = 100;
		this.L = 50;
		this.X = 10;
		this.V = 5;
		this.I = 1;
	}

}

function stringMultiplier(str, n) {
	var result_str = "";

	for (var i = 0; i < n; i++) {
		result_str = result_str + str;
	}

	return result_str;
}


function numOfSymbol(num, symbol) {
	var roman_dict = new RomanDict();

	// console.log([parseInt(num / roman_dict[symbol]), num % roman_dict[symbol]]);
	return [parseInt(num / roman_dict[symbol]), num % roman_dict[symbol]];
}


function convertToRoman(num) {
	var roman_dict = new RomanDict();
	var result_str = "";
	var temp_num = num;

	for (var property in roman_dict) {
    	if (roman_dict.hasOwnProperty(property)) {
    		result_str = result_str + stringMultiplier(property, numOfSymbol(temp_num, property)[0]);
			temp_num = numOfSymbol(temp_num, property)[1];
    	}
	}

	// ETL for XI and VI etc...
	result_str = result_str.replace("DCCCC", "CM");
	result_str = result_str.replace("CCCC", "CD");
	result_str = result_str.replace("LXXXX", "XC");
	result_str = result_str.replace("XXXX", "XL");
	result_str = result_str.replace("VIIII", "IX");
	result_str = result_str.replace("IIII", "IV");

	// console.log(result_str);
	return result_str;
}

// convertToRoman(36);

console.log(convertToRoman(2) === "II")
console.log(convertToRoman(3) === "III")
console.log(convertToRoman(4) === "IV")
console.log(convertToRoman(5) === "V")
console.log(convertToRoman(9) === "IX")
console.log(convertToRoman(12) === "XII")
console.log(convertToRoman(16) === "XVI")
console.log(convertToRoman(29) === "XXIX")
console.log(convertToRoman(44) === "XLIV")
console.log(convertToRoman(45) === "XLV")
console.log(convertToRoman(68) === "LXVIII")
console.log(convertToRoman(83) === "LXXXIII")
console.log(convertToRoman(97) === "XCVII")
console.log(convertToRoman(99) === "XCIX")
console.log(convertToRoman(500) === "D")
console.log(convertToRoman(501) === "DI")
console.log(convertToRoman(649) === "DCXLIX")
console.log(convertToRoman(798) === "DCCXCVIII")
console.log(convertToRoman(891) === "DCCCXCI")
console.log(convertToRoman(1000) === "M")
console.log(convertToRoman(1004) === "MIV")
console.log(convertToRoman(1006) === "MVI")
console.log(convertToRoman(1023) === "MXXIII")
console.log(convertToRoman(2014) === "MMXIV")
console.log(convertToRoman(3999) === "MMMCMXCIX")
```

#### Screenshot

<img width="915" alt="screen shot 2016-06-11 at 11 53 39 pm" src="https://cloud.githubusercontent.com/assets/5828418/15985982/c5bbe3bc-302f-11e6-9ba0-32ae7a4cabfa.png">

**Comment by robjloranger** [comment]
your code works fine for me in FCC 'ide'
also tested running through node. 4 results in IV and type is string

**Comment by erictleung** [comment]
@savourylie you code passes just fine for me on the FCC website. But I'm a bit unclear on what you mean between the FCC "IDE" and just Chrome and Node.

**Comment by robjloranger** [comment]
@erictleung Not sure about Chrome, I assume running local file and viewing in dev console?
Node however, if you install node js you can execute js at the command line.

**Comment by savourylie** [comment]
@robjloranger Thanks for clarifying for me. Yes that's what I meant by running in Chrome (dev tools console). I usually test my JS in node.js which allows me to test my code "headlessly". I find that very convenient. :)

Oh and strangely, when I opened up FCC just now, everything was running green! So I pressed Test/Run button again and it showed all passed and jumped to the next challenge.

I don't know what happened as I didn't change my code nor anything. But I guess this mean the problem is fixed now. Thanks guys. Deeply appreciated!
